tests/e2e/support/commands.js

```diff
@@ -806,19 +806,23 @@ Cypress.Commands.add('discardOpponent', (card1, card2) => {
       if (card2) {
         [cardId2] = getCardIds(game, [card2]);
       }
-      io.socket.get(
-        '/game/resolveFour',
-        {
-          cardId1,
-          cardId2,
-        },
-        function handleResponse(res, jwres) {
-          if (!jwres.statusCode === 200) {
-            throw new Error(jwres.body.message);
-          }
-          return jwres;
-        },
-      );
+        io.socket.get(
+          '/game/resolveFour',
+          {
+            cardId1,
+            cardId2,
+          },
+          function handleResponse(res, jwres) {
+            try {
+              if (jwres.statusCode !== 200) {
+                throw new Error(jwres.body.message);
+              }
+              return jwres;
+            } catch (err) {
+              return err;
+            }
```

Comment:
I had to add the try catch here, because the test in the 4's spec that checks to make sure you're opponent cant resolve a four with no cards throws an error and cypress will fail on uncaught errors

Reply:
Makes sense!

Reply:
I think this is a reasonable way to do it w/o affecting the general error handling strategy, which is nice